src/utils/removeObjectKey.js

```diff
@@ -4,6 +4,6 @@ export default function removeObjectKey ( generator, node, key ) {
 	const index = properties.findIndex( property => property.key.type === 'Identifier' && property.key.name === key );
 	if ( index === -1 ) return;
 	const a = properties[ index ].start;
-	const b = index < properties.length - 1 ? properties[ index + 1 ].start : properties[ index ].end;
+	const b = index < properties.length - 1 ? properties[ index + 1 ].start : node.end - 1;
```

Comment:
I *assume* that `node.end` is always going to be the location of the `}`. It was even when there were comments directly touching the `}`, and I don't know what other edge cases would be.